index.html

```diff
@@ -126,7 +126,58 @@
 				</p>
 			</div> <!-- cd-faq-content -->
 		</li>
-	</ul> <!-- cd-faq-group -->
+	</ul> 
+        <ul id="frameworki" class="cd-faq-group">
+                <li class="cd-faq-title"><h2>Frameworki</h2></li>
+                <li>
+                        <a class="cd-faq-trigger" href="#0">Dlaczego hejtujemy Laravela?</a>
+                        <div class="cd-faq-content">
+                                <p>
+                                        Ja, piszący te słowa, pracowałem przez dwa miesiące z Laravelem i mója do niego niechęć nie opiera się na zawiści wobec Taylora Otwella.
```

Comment:
Tylko, że FAQ PHPers to nie "prywatne odczucia Maćka" a raczej źródło informacji, bez sprecyzowanego autora.